cves/kernel/CVE-2017-16913.yml

```diff
@@ -55,7 +59,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: CVE-2017-16913 is a denial of service vulnerability in the USB over IP subsystem. Prior to the patch the CMD_SUBMIT codepath had insufficient checks on the size (https://elixir.bootlin.com/linux/v4.14.7/source/drivers/usb/usbip/stub_rx.c#L471). Because of this, a threat actor could potentially send an unlimited size resulting in a out-of-memory error. In addition, there were also some vulnerabilities in the isochronous transfer type where a threat actor would be able to request an extremely large number buffer (https://elixir.bootlin.com/linux/v4.14.9/source/drivers/usb/usbip/stub_rx.c#L360 - transfer_buffer_length checked) or an extremely large number of packets (https://elixir.bootlin.com/linux/v4.14.9/source/drivers/usb/usbip/stub_rx.c#L397 - number of packets are checked against transfer_buffer_length).Any of these situations could cause the USB over IP subsystem to perform a large allocations which could lead to a denial of service. 
```

Comment:
The references are okay, but just make the rest of your description harder to read. I would suggest removing them or adding them to the bottom of the description.

Reply:
I definitely agree. I removed the references.